drivers/lora/sx1276.c

```diff
@@ -42,30 +48,25 @@ static struct sx1276_dio sx1276_dios[] =
 #define SX1276_MAX_DIO ARRAY_SIZE(sx1276_dios)
 
 struct sx1276_data {
```

Comment:
The `dev_data` should probably be static.

Reply:
okay